.github/workflows/ariane-scheduled.yaml

```
@@ -16,6 +16,8 @@ permissions:
 
 jobs:
   ariane-scheduled:
+    # Skip running workflow on forks
+    if: github.repository_owner == 'cilium'
```

I wonder if this would be easier to opt in/out with something like this:

```suggestion
    if: ${{ always() && env.ENABLE_CILIUM_CI == 'true' }}
```

Then anyone who wants to use the CI can either (a) edit the workflow on their own branch to remove the check, or (b) set a repository-wide variable for `ENABLE_CILIUM_CI` and set the value to `true` to enable all workflows. This would help for others who run these workflows outside this organization.

I'm also open to alternative proposals from others in the community. I know this has been a bit of a pain point in the past and we've primarily relied on contributors disabling the workflows through the 'Settings' menu on their fork (which is what GitHub recommends, as far as I understand).

---

@joestringer, thanks for your suggestion. 

That’s definitely a valid approach as well!

For completeness, the suggested configuration would involve adding the GitHub environment variable ENABLE_CILIUM_CI to the Cilium project, which might require support from a maintainer.

In my opinion, both options 

- `${{ always() && vars.ENABLE_CILIUM_CI == 'true' }}` 
and 
- `github.repository_owner == 'cilium'` 

works. They both disable CI runs on forks by default while allowing them to be explicitly enabled, either through a simple search-and-replace or via new GitHub environment settings.

If there’s a strong preference from the project maintainers or the community for one approach over the other, I’m happy to adjust and move forward accordingly.

---

I've reached out to some others in case there are other opinions, but from my perspective the `ENABLE_CILIUM_CI` environment variable approach would work well, and I'd be happy to facilitate that. I know there are other forks regularly running the full CI, so this seems easier to manage rather than requiring such forks to modify each workflow. As you say, that still leaves multiple options open for developers who want to use some subset of the workflows in the tree on their own branches.

---

@joestringer @ishuar I like the `ENABLE_CILIUM_CI` proposal. I've added the `ENABLE_CILIUM_CI` variable with the value `true` so we can use it right away.

---

@aanm & @joestringer 

Searched & Replaced `github.repository_owner == 'cilium'` with `always() && vars.ENABLE_CILIUM_CI == 'true'` , However, there were places where `always()` could be avoided, but even having it should not be an issue. 

Thanks for the feedback, and let me know what you think about the latest proposed changes.